### ISSUE TITLE
Test suite fails with Twisted 18.4.0

### ISSUE THREAD

**Issue by mcepl** [twisted/epsilon#25]
Happens five times due to the same error.
```
======================================================================
ERROR: testFormatterWithoutPartitionStats (epsilon.test.test_benchmark.ReporterTestCase)
testFormatterWithoutPartitionStats
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/twisted/internet/defer.py", line 150, in maybeDeferred
    result = f(*args, **kw)
  File "/usr/lib64/python2.7/site-packages/twisted/internet/utils.py", line 201, in runWithWarningsSuppressed
    reraise(exc_info[1], exc_info[2])
  File "/usr/lib64/python2.7/site-packages/twisted/internet/utils.py", line 197, in runWithWarningsSuppressed
    result = f(*a, **kw)
  File "/home/abuild/rpmbuild/BUILD/Epsilon-0.7.1/epsilon/test/test_benchmark.py", line 215, in testFormatterWithoutPartitionStats
    56), False, False))
  File "/home/abuild/rpmbuild/BUILD/Epsilon-0.7.1/epsilon/scripts/benchmark.py", line 380, in formatResults
    twisted_version = twisted.version._getSVNVersion()
AttributeError: 'Version' object has no attribute '_getSVNVersion'

```

**Comment by mithrandi** [comment]
This was fixed by #24, but we need to do a new release including that change.

**Comment by mithrandi** [comment]
0.7.2 is released now.
